app/views/gardens/_form.html.haml

```diff
@@ -16,15 +16,18 @@
   .form-group
     = f.label :description, :class => 'control-label col-md-2'
     .col-md-8
-      = f.text_area :description, :rows => 6, :class => 'form-control', :placeholder => 'optional'
+      = f.text_area :description, :rows => 6, :class => 'form-control'
+      %span.help-block.optional
```

Comment:
Fair point

On Fri, May 27, 2016 at 9:23 AM pozorvlak <notifications@github.com> wrote:

> In app/views/gardens/_form.html.haml
> <https://github.com/Growstuff/growstuff/pull/922#discussion_r64903340>:
>
> > @@ -16,15 +16,18 @@
> >    .form-group
> >      = f.label :description, :class => 'control-label col-md-2'
> >      .col-md-8
> > -      = f.text_area :description, :rows => 6, :class => 'form-control', :placeholder => 'optional'
> > +      = f.text_area :description, :rows => 6, :class => 'form-control'
> > +      %span.help-block.optional
>
> These two lines get repeated a lot - perhaps factor them out into a helper
> or partial?
>
> —
> You are receiving this because you authored the thread.
>
> Reply to this email directly, view it on GitHub
> <https://github.com/Growstuff/growstuff/pull/922/files/079cecc529cbd253a0ac82efea6683f3638c55be#r64903340>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAYfcNa4VR7-0RA4BSVnvsGQl0EL4Ok6ks5qFvBagaJpZM4In8Zs>
> .
>